sdk/python/feast/infra/utils/snowflake_utils.py

```diff
@@ -53,13 +53,20 @@ def get_snowflake_conn(config, autocommit=True) -> SnowflakeConnection:
     else:
         kwargs = {}
 
+    if "schema" in kwargs.keys():
+        kwargs["schema_"] = kwargs.pop("schema")
+
     kwargs.update((k, v) for k, v in config_dict.items() if v is not None)
     [
         kwargs.update({k: '"' + v + '"'})
```

Comment:
just noticed, you should be able to this here:
```
kwargs[k] = f'"{v}"' 
```

Reply:
not quite following here

Reply:
there is specific keywords that need quotes around them, others dont require

Reply:
sorry, I just meant that instead of doing 
```
kwargs.update({k: '"' + v + '"'})
```

you can do the thing I suggested - you can still do it selectively for the keywords that need quotes, just that assignment style seems more pythonic than the `.update` style